addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -117,10 +117,8 @@
                          <img border="0" src="/mass_mailing/static/src/img/theme_basic/s_default_image_logo.png" style="height:auto;max-width:400px;" alt="Your Logo" />
                     </a>
                 </div>
-                <div class="col-lg-4 text-right o_mail_no_resize">
-                    <div class="o_mail_header_social">
-                        <t t-call="mass_mailing.social_links"/>
-                    </div>
+                <div class="col-lg-4 text-right o_mail_no_resize o_mail_header_social">
```

Comment:
About the commit message: What do you mean by remove "on" block ? Would the meaning be different to you if it were "remove block for social links" ? I'm genuinely trying to understand.

Reply:
Indeed, "on" was a mistake